### ISSUE TITLE
predict produce same label for every line

### ISSUE THREAD

**Issue by jesuiskelly** [mlampros/fastText#2]
Hi there,

Thanks for creating and making the fastText library open source. I was trying out your fastText package and stumbled upon an issue. I wonder if you can help?

Essentially, I used the cooking data I downloaded from Facebook, created and test and training sets. I trained a model and used it to predict labels for test data. When I checked the predicted labels, every one of them says "__label__bread".

I find that very puzzling. I wonder if I have perhaps done something wrong?

I have attached the r script and relevant data files for you to have look if that's alright.

Thank you so much for your help in advance.
[rstudio-export (1).zip](https://github.com/mlampros/fastText/files/5560158/rstudio-export.1.zip)

**Comment by mlampros** [comment]
Hi @jesuiskelly,

what I most of the times do before I use my own data is to try to reproduce existing results. Did you try to reproduce my results in the [blog post](http://mlampros.github.io/2019/04/11/fastText_updated_version/)? If this worked for you then you have to adjust your new data to this format to receive the expected predictions (you can find the data that I've used in the blog post in [this weblink](https://github.com/mlampros/DataSets/blob/master/fastText_data.zip)).

**Comment by jesuiskelly** [comment]
@mlampros, thanks for getting back to me.

Yes, I managed to resolve this problem by comparing your files with mine:
The input files I have come from the same source as yours (stackexchange). What went wrong was when I split the cooking.stackexchange.txt file into test and train sets, I used the write.table function, which by default added quotes to each line of the data. This turns each line into a string. FastText, therefore, failed to produce a model. This was fixed by setting quote as F in the write.table function.

Thanks again for your help!
